assets/src/edit-story/elements/text/index.js

```diff
@@ -58,6 +58,8 @@ export const resizeRules = {
   vertical: false,
   horizontal: true,
   diagonal: true,
+  minWidth: 20,
+  minHeight: 20,
```

Comment:
The text's `minHeight` should be entirely up to `font-size`. E.g. if we allow e.g. `font-size == 10`, then, we have to allow `minHeight < 20`. Can we perhaps use `null/undefined` here to indicate that `minHeight` is not enforced?

Reply:
`undefined` will also work (producing `NaN` now), but maybe `0` will be more semantic?